src/tls/cockpit-certificate-helper.in

```diff
@@ -56,6 +56,17 @@ cmd_ipa_request() {
     install_key 10-ipa.key
 }
 
+cmd_ipa_cleanup() {
+        # clean up keytab
+        if [ -e "${KEYTAB}" ]; then
+            ipa-rmkeytab -k "${KEYTAB}" -p "${SERVICE}"
+        fi
+
+        # clean up certificate
+        ipa-getcert stop-tracking -f 10-ipa.cert -k 10-ipa.key
+        rm -f "${COCKPIT_WS_CERTS_D}"/10-ipa.*
```

Comment:
Can you please spell out .key and .cert here instead of the glob? We don't want this to remove backup files, nor globbing.

Reply:
OK.